cmd/podman/common/specgen.go

```diff
@@ -248,6 +249,10 @@ func FillOutSpecGen(s *specgen.SpecGenerator, c *ContainerCLIOpts, args []string
 		}
 	}
 
+	if !registry.IsRemote() && c.UserNS == "" {
```

Comment:
What if the user explicitly set `--userns=""` - is there any way to do this higher up in the stack, where we know whether the flag was changed?